src/main/steps/applicant/contact-preferences/contact-post/contest.test.ts

```diff
@@ -20,15 +20,17 @@ const en = {
 };
 
 const cy = {
-  caption: `Case number - welsh #${caseNumber}`,
-  title: 'Contact Preferences -welsh',
-  subTitle: 'Personal details -welsh',
-  continue: 'Submit - welsh',
+  caption: `Rhif yr achos #${caseNumber}`,
+  title: 'Dewisiadau Cyswllt',
+  subTitle: 'Manylion personol',
+  continue: 'Cyflwyno',
   textList: [
-    'You have decided to receive updates by post. -welsh',
-    'You will no longer receive updates by email. You can still access previous updates through your dashboard. -welsh',
+    'Rydych wedi penderfynu cael diweddariadau drwy’r post.',
```

Comment:
Does it require post at the end?

Reply:
Yes, it's in the translation given in the excel document